portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -39,27 +40,42 @@
   
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    Query query = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
+    String defaultCommentLimit = "5";
+    int commentLimit = Integer.parseInt(getParameter(request, "numShown", defaultCommentLimit));
```

Comment:
not a great parameter name, best to avoid capitalization.  Why not just limit

Reply:
Agreed mediocre name; renamed to commentLimit for (extra) clarity

Reply:
Renamed to "commentLimit" for consistency